<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Анализ кода</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="analyseButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Проанализировать</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSLOC">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelEL">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelComments">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelOnlyCode">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelOperators">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelOperands">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTOpertators">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTotalOperands">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelLengthVoc">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelLength">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelCyclomatic">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelCI">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
